psh/test-auth.py

```diff
@@ -75,7 +75,7 @@ def harness(p):
     assert_auth(p)
     p.send(printables + '\n')
     assert p.expect_exact(['Login:', pexpect.TIMEOUT]) == 0, 'Long login does not repeat login procedure'
-    logintools.assert_login_fail(p, cred_ok.user, printables)  # too long password
+    logintools.assert_auth_fail(p, cred_ok.user, printables)  # too long password
 
     # Test backspace
     assert_auth(p)
```

Comment:
There is a one second timeout:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/e446f8cc-e0cc-4eb9-b885-8e64bfac898a)